Pocket/pocket-ios
# IN-650 Add Syndicated articles to progress recording

## Summary
UITests have been crashing due to a fatalError included in the RecommenationViewModel

## Test Steps
Open a syndicated article scroll down to a point
Close the article
Reopen the article
The view should scroll to around the point you left off
